ui/app/api/peers/create/route.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CreatePeerRequest,
+  CreatePeerResponse,
+  CreatePeerStatus,
+} from '@/grpc_generated/route';
+import { GetFlowServiceClient } from '@/rpc/rpc';
+import { constructPeer } from '../util';
+
+export async function POST(request: Request) {
```

Comment:
can we move this to `peers/route.ts` and service both validate and create based on a param in the json or something else?